index.js

```diff
@@ -0,0 +1,56 @@
+//set requires
+const express = require("express");
+const knex = require("knex");
+const cors = require("cors");
+const bcrypt = require("bcryptjs")
+
+const dbConfig = require("./knexfile");
+const db = knex(dbConfig.development);
+
+//set port #
+const PORT = 8000;
+
+const server = express();
+server.use(express.json(), cors());
+
+/*Creates a user using the information sent inside the body of the request. Hash the password before saving the user to the database.*/
+server.post("/api/register", (req, res) => {
+   const user = req.body;
+   if (user.username && user.password) {
+   user.password = bcrypt.hashSync(user.password, 14);
+   db("users").insert(user)
+      .then(ids => {
+         res.status(201).json({ids: ids})
```

Comment:
```suggestion
         res.status(201).json({ ids })
```

This is just another simplified form of doing it.